added button in storybook + commented styles

Ali-Khamis opened the pull request:
## description
1- added buttons using react and storybook
2- commented styles in stories/examples/(header.css & page.css) because it targets elements which effect everyone's code
*Work done after reviewed
1- added sizes and controls for the toggler  as requested
2- fixed the fonts and spaces of buttons  as requested
3-  deleted unused commented code
4- changed my storybook folder structure
5- added text input with no label  ( width 100%  ) 
6-  added image upload button (width 100%  )
7-deleted duplicated files
8-added a control for the Input text border radius
*Work done after second review 
1- General changes : changed the radio controls to be inline-radio as recommended 
2- Input changes : removed the icon when the input is disabled 
3-Toggler changes : fixed the size of the small toggler to match the design

Ali-Khamis committed:
added button in storybook + commented styles

Ali-Khamis committed:
added avatar + toggler + moved button into folder

Ali-Khamis committed:
Merge branch 'master' into feature/storybook-test-buttons

Safei-Ashraf reviewed (changes requested):
Great Work!
Small notes though:
1-storybook/src/Ali/TextButton/Button.js  >>> Button font weights, spacing and height do not match design, if a certain padding class does not exist in the config you could use the TW classes, they are not overridden and fully accessible to us.

2-storybook/src/Ali/Toggler/Toggler.js  >>>> does not match design, review sizing and spacing, also note there are 2 sizes in the design you need to match them both.
3-storybook/src/Ali/Toggler/Toggler.stories.js >>>>>>>> consider adding storybook controls to view what it would be like when the button is disabled, also another control to switch sizes from BIG to SMALL once you fix it in the component itself.

Ali-Khamis committed:
deleted unused commented code + fixed the fonts and spaces of buttons + added sizes and controls for the toggler

Ali-Khamis committed:
Merge branch 'master' into feature/storybook-test-buttons

Ali-Khamis committed:
Merge branch 'master' into feature/storybook-test-buttons

Ali-Khamis committed:
changed my folders structure +
added text input with no label  ( width 100% not hardcoded ) + added image upload button (width 100% not hardcoded )

Ali-Khamis committed:
deleted dublicated files

Ali-Khamis committed:
Merge branch 'master' into feature/storybook-test-buttons

Ali-Khamis committed:
added a control for the border radius

Safei-Ashraf reviewed (changes requested):
1- toggler small size does not match the design. resize the white ball maybe and check to space properly just a small tweak.
2- text input field, when disabled should not show the icons inside of it (error - success) they appear stuck on top of it.
![image](https://user-images.githubusercontent.com/44810632/105188010-1c8b7900-5b3c-11eb-9fad-4a95c1a02d77.png)

3-consider adding argTypes into your stories so you get full control over how controls would be displayed on storybook 
![image](https://user-images.githubusercontent.com/44810632/105188313-71c78a80-5b3c-11eb-82b8-80fa3b0fdb40.png)
**Recommend** using "inline-radio" as your way to go with options,  if you need live examples just refer to my code inside src or Sahl's we have multiple cases applied. and ofc the docs would be nice.
None of your stories has fully controlled "controls" so give it a try in this one or the next PR.

Ali-Khamis committed:
changed requested

Safei-Ashraf reviewed (approved):
Looking عظمة  to me.

in the future avoid adding dev related controls into your story e.g: className, html attrbs, or code related stuff like 
![image](https://user-images.githubusercontent.com/44810632/105199455-734a8000-5b47-11eb-82f3-812e9cf7c2d1.png)
